spec/declaration.dd

```diff
@@ -296,12 +298,29 @@ alias myint = abc.Foo.bar;
         )
 
 --------------------
-alias myint = int;
+alias MyInt = int;
 
 void foo(int x) { ... }
-void foo(myint m) { ... } // error, multiply defined function foo
+void foo(MyInt m) { ... } // error, multiply defined function foo
+--------------------
+
+        $(P
+        Type aliases can sometimes look indistinguishable from
+        other symbol aliases:
+        )
+
+--------------------
+alias abc = foo.bar; // is it a type or a symbol?
 --------------------
 
+        $(P
+        The distinction is made in the semantic analysis pass.
```

Comment:
I Don't think this should be mentioned in the spec.

Reply:
This was an existing sentence, I've now removed it.